package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-config-edx",
+  "version": "0.0.1",
+  "description": "ESLint config for edX JavaScript code.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint ."
```

Comment:
I was thinking about this line, and how we could test the rules. One idea I had is to include a 'samples' directory with code demonstrating each of the rules we care about. Then we could have CI actually run eslint and fail if any of the samples have errors. What do you think of that?

It would also be good to have some negative tests, but I don't know how you could achieve that. Maybe you could have a separate directory for code with issues, and then run eslint twice. The bad examples would have an expected number of eslint failures.

Are there any good examples of how to write tests for eslint rules?